src/release_notes_workflow/release_notes.py

```diff
@@ -19,18 +19,19 @@
 
 class ReleaseNotes:
 
-    def __init__(self, manifest: InputManifest, date: str, action_type: str) -> None:
-        self.manifest = manifest
+    def __init__(self, input_manifests: List[InputManifest], date: str, action_type: str) -> None:
+        self.manifests = input_manifests  # type: ignore[assignment]
         self.date = date
         self.action_type = action_type
 
     def table(self) -> MarkdownTableWriter:
         table_result = []
-        for component in self.manifest.components.select():
-            if component.name == 'OpenSearch' or component.name == 'OpenSearch-Dashboards' or component.name == 'notifications-core':
```

Comment:
I think we do want to ignore OS/OSD core because core has their own format and link reference isnt it?
cc: @bbarani

Reply:
I can keep it for `check` and ignore at `compile`.

Reply:
Yeah that sounds better. We for sure need it for check but can ignore for compile.

Reply:
Done